tests/functionnal/Document.php

```diff
@@ -423,7 +425,7 @@ public function testCanViewKnowbaseItemFile() {
          $kbItem->update(
             [
                'id'     => $kbItem->getID(),
-               'is_faq' => true,
```

Comment:
mysql has never been supported booleans, maybe should they be converted at higher level (I'm pretty sure there will be plenty of those kind of dumb changes).

Reply:
done